.github/workflows/docker.yml

```diff
@@ -1,4 +1,26 @@
 name: "Docker Build"
+env:
+  repository: "cloudcustodian"
```

Comment:
can we move this default to the composite input's default value and thus remove it from env vars, which are mutable, to either default or explicitly passed. alternatively remove the env var here and explicitly set it as a string in the matrix job below.